recipes/pydoe/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.3.8" %}
+
+package:
+  name: pydoe
+  version: {{ version }}
+
+source:
+    fn: pyDOE-afc5949f01c010a26764adba9624871d51797270.tar.gz
+    url: https://github.com/tisimst/pyDOE/archive/afc5949f01c010a26764adba9624871d51797270.tar.gz
+    sha1: 54f87d106c3f49e1b4cff17a31ef8a9e0142ec5b
+
+build:
+  script: python setup.py install
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scipy
+
+  run:
+    - python
+    - scipy >=0.13
+    - numpy >=1.8
+
+test:
+  imports:
+    - pyDOE
+
+about:
+  home: https://github.com/tisimst/pyDOE
+  license: BSD License (3-clause)
+  summary: The pyDOE package is designed to help the scientist, engineer, statistician, etc., to construct appropriate experimental designs.
```

Comment:
Might be good if we could shorten this a bit. A longer form is ok as a `description`.